blocks/cards/cards.js

```diff
@@ -76,22 +78,63 @@ function fetchAPIBasedCards(cardInfos, apis) {
   });
 }
 
-async function fetchRuleBasedCards(config, cardInfos, idx) {
-  const blogs = await getLocaleBlogs();
-  switch (toClassName(config.rule)) {
-    case 'home-page-latest':
-      cardInfos[idx] = blogs.slice(0, 3);
-      break;
-    case 'home-page-category':
-      cardInfos[idx] = FILTERS.category(
-        // skip first 3 blogs as they are used for the latest section
-        blogs.slice(3),
-        toClassName(config.category),
-      ).slice(0, 3);
+function homepageLatestRule(blogs, cardInfos, idx) {
+  cardInfos[idx] = blogs.slice(0, 3);
+}
+
+function homepageCategoryRule(blogs, cardInfos, idx, config) {
```

Comment:
Is there a scenario where the top 3 articles are not the most "recent" one? I am thinking someone correcting an article that has initially been published in the past.

Reply:
The authoring experience looks good to me and we don't have to change it. But, as an alternative we could potentially consider the following experience: 

+---------------------------+
| Cards (Sidebar, Featured)           |
+--------+------------------+

### Trends & Research

+--------------------------------------+
| Cards (Sidebar, Trends and Research )                     
+--------+-----------------------------+

Reply:
@sergiucoman 
> Is there a scenario where the top 3 articles are not the most "recent" one? I am thinking someone correcting an article that has initially been published in the past.

my understanding is that no, that scenario cannot happen.

>isn't the index sorted by publication date at any rate (and that's manually maintained and doesn't change when an article is corrected)?

It's how @benpeter mentions here, if I understood correctly. The initial publication date metadata remains unchanged on corrections. What they do is add a `Updated ${date of update}` in the text to mark corrections. I've found this example: https://www.servicenow.com/blogs/2020/covid-business-resources.html

Reply:
> The authoring experience looks good to me and we don't have to change it. But, as an alternative we could potentially consider the following experience

That's true, we could've opted for that experience as well. I tried to keep the authoring consistent with the blog list and the cards on the homepage, which require more configuration than just the rule itself.

the way that I approach it in my mind is:
* block variation - what is in parentheses Cards (Sidebar) - I use when I need to write different CSS based on the class that gets generated .sidebar
* block configuration - I use when I need the content from the block to change, but styling to remain consistent

In the end, both experiences are equally valid from my point of view, just that having already implemented two instances of this one, I tried to make it consistent.